tools/test-popular-containers/build_rootfs.sh

```diff
@@ -73,9 +73,9 @@ passwd -d root
 EOF
 }
 
+make_rootfs amazonlinux:2023
```

Comment:
why does this fix it? Can't the same issue happen in `make_rootfs ubuntu:latest`?

Reply:
added